pkg/definition/gen_sdk/gen_sdk.go

```diff
@@ -509,3 +547,63 @@ func newModifierOnLanguage(lang string, generator *Generator) Modifier {
 		panic("unsupported language: " + lang)
 	}
 }
+
+// getValueType returns the openapi type of the value
+func getValueType(i interface{}) CUEType {
```

Comment:
Shall we move this Function to Utils package?

Reply:
This function is newly introduced. Let's hold it here until possible reuse occurs.

Reply:
> This function is newly introduced. Let's hold it here until possible reuse occurs.

ok